lib/srv/alpnproxy/local_proxy.go

```diff
@@ -378,6 +378,27 @@ func (l *LocalProxy) getCerts() []tls.Certificate {
 	return l.cfg.Certs
 }
 
+// CheckCertExpiry checks the proxy certificates for expiration and runs given checking function.
+func (l *LocalProxy) CheckCert(checkCert func(cert *x509.Certificate) error) error {
+	l.cfg.Log.Debug("checking local proxy certs")
+	l.certsMu.RLock()
+	defer l.certsMu.RUnlock()
+	if len(l.cfg.Certs) == 0 {
+		return trace.NotFound("local proxy has no TLS certificates configured")
+	}
+	cert, err := utils.TLSCertLeaf(l.cfg.Certs[0])
```

Comment:
Looks like you are right, we are never using more than one cert. Adjusted it to be singular across the board.

FWIW I think the original reason was to make a one-to-one translation to `tls.Config.Certificates`, but I think the verbosity is not worth it since it's not actually used pluraly.